browsergym/experiments/src/browsergym/experiments/loop.py

```diff
@@ -657,6 +658,75 @@ def summary_info(self) -> dict:
                 self._summary_info = json.load(f)
         return self._summary_info
 
+    def get_tape(self) -> Tape:
+        """
+        Return experiment trace in a format compatible with the TapeAgents framework
+        """
+        tape_steps = []
+        for step_info in self.steps_info:
+            # extract observation step
+            if step_info.obs is not None:
+                screenshot: str = ""
+                screenshot_som: str = ""
+                if "screenshot" in step_info.obs:
+                    screenshot = f"{self.exp_dir}/screenshot_step_{step_info.step}.png"
+                    step_info.obs.pop("screenshot")
```

Comment:
this is also popping in the stored self.steps_info, which could lead to unexpected errors if someon tries to use results after calling get_tape